src/client/App.jsx

```diff
@@ -33,7 +33,7 @@ window.onload = () => {
   Iso.bootstrap((state, container) => {
     alt.bootstrap(state);
 
-    const appHistory = useScroll(useRouterHistory(createBrowserHistory))();
+    const appHistory = (useRouterHistory(createBrowserHistory))();
```

Comment:
By the way, I removed the `useScroll` functionality because it's annoying when you click on the next/prev button and it takes you to the top of the page. It affects the whole app so we will have to test it out and see if anyone gets annoyed by it.